### DISCUSSION TITLE
blog/understanding-iptables/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#282] [Announcements]
# blog/understanding-iptables/

本文将为大家简单介绍下 iptbles，其中的表及执行顺序。

https://jimmysong.io/blog/understanding-iptables/

**Comment by wanejen** [comment]
表链规则对应错了，raw表只有PREROUTING和OUTOUT链，nat表没有INPUT链。

**Reply by rootsongjc** [reply]
感谢反馈，已修改。

**Reply by FireSoilD** [reply]
新版 nat 表增加了 INPUT 链

**Reply by rootsongjc** [reply]
是从哪个版本开始增加的？

**Reply by FireSoilD** [reply]
具体版本不清楚，当时用Centos6还没有，Centos7就有了，我看了下 1.4.21 这个版本（Centos7）就有 INPUT

**Comment by funlake** [comment]
大佬，博客标题的iptables拼错了

**Reply by rootsongjc** [reply]
感谢反馈，已经改过来了。
